scripts/check-diff-line-width.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2022 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+git diff master -U0 cdc pkg |
```

Comment:
What happens on the release branch?

Also, I get check-leaktest-added effects when I test locally and it may not get run?(check-leaktest-added will terminate the check)

Reply:
This check is filtered when it bases on a release branch, see makefile.